### DISCUSSION TITLE
Turtlebot4 Network Setup

### DISCUSSION THREAD

**Discussion by zzzghm** [turtlebot/turtlebot4#482] [Q&A]
Robot Model
**Turtlebot4 Standard**

ROS distro
**Humble**

Networking Configuration
**Simple Discovery/ Discovery Server**

OS
**Ubuntu 22.04**

Built from source or installed?
**Built from Source**

Type of issue
**Networking, I try to set up the network for the new arrival turtlebot4, after set the wifi to Client, it shows 'UNKNOWN' ip with only two light on, and the green light on the raspberry pi never turned on(shown in figure).**
![IMG_9870](https://github.com/user-attachments/assets/7d8a8395-c109-44e7-86ec-d6670b370e5e)
![IMG_9871](https://github.com/user-attachments/assets/f0ac86f8-6e25-4ea0-abc2-0784be20482c)

**I tried all the methods I can find online several times, like check the cables connection, flash the raspberry pi image(both 1.0.0 and 1.0.4), factory reset the create3, connect to different network, reboot, install newest Humble version(2.6). None of this works!!!!!!!!
I've been struggle it for almost one week with the extremely simple set up.
Can you help me solve it ASAP?**

**Comment by zzzghm** [comment]
FYI: flash the raspberry pi image 1.04, it will be a totally black screen with no lights on, so go back to 1.0.0 will have the above image

**Comment by hilary-luo** [comment]
@zzzghm to get quicker responses from our team enter a troubleshooting issue (those are actively monitored). It looks like in at least one of those cases you may be flashing the "lite" image on a standard robot. The lite does not have a screen and thus will not activate the screen or lites. In the other situation it looks like the create3 likely has not been updated to the same version as the raspberry pi and thus functionality that relies on the create3 is down. Feel free to open a troubleshooting issue if you are still running into troubles. Generally when the IP generates as unknown it is because the credentials were entered incorrectly.
